dev/idm-am-common-user/amster/config/realms/root/OpenDJ/OpenDJ.json

```diff
@@ -0,0 +1,60 @@
+{
```

Comment:
Are you sure you need this file? I removed it from my sample recently - instead, favoring the options available in the default amster install command:
```
    --userStoreDirMgr "{{ .Values.userStore.dirManager }}" \
    --userStoreDirMgrPwd  {{ .Values.userStore.password }} \
    --userStoreHost {{ .Values.userStore.host }} \
    --userStorePort {{ .Values.userStore.port }} \
    --userStoreRootSuffix {{ .Values.userStore.suffix }} \
    --userStoreType {{ .Values.userStore.storeType }} \
```

Reply:
See https://github.com/ForgeRock/forgeops-init/pull/146